### PR TITLE
Polyfills

### PR DESCRIPTION
**Pull request by luizbaldi**
Adding translation for "Polyfills" :)

Feel free to share thoughts on the translation and let me know if any changes are needed!

### CODE REVIEW

File: 1-js/03-code-quality/06-polyfills/article.md
Comment (odsantos) [inline 1-js/03-code-quality/06-polyfills/article.md]: Please...
'...as *features* (funcionalidades) suportadas...'

File: 1-js/03-code-quality/06-polyfills/article.md
Comment (odsantos) [inline 1-js/03-code-quality/06-polyfills/article.md]: Please...
'...tornar o código compatível para navegadores...'

**Comment by odsantos** [comment]
Would you please also notice that the foreign language terms (English) are, in general, emphasized? But not JavaScript. Thanks.

**Comment by luizbaldi** [comment]
@odsantos Thanks for your review! I've just finished pushing the changes, hope you like it :)

Let me know if any more change is needed!

**Comment by odsantos** [comment]
Hope to do it as soon as possible. Thanks.  
 
  On Mon, Oct 14, 2019 at 1:54 AM, Luiz Baldi<notifications@github.com> wrote:   
@luizbaldi requested your review on: #71 Polyfills.

File: 1-js/03-code-quality/06-polyfills/article.md
Comment (odsantos) [inline 1-js/03-code-quality/06-polyfills/article.md]: Please,
 - 'Então, precisamos de configurar o transpiler ...'
- do not italicize the parentheses.

File: 1-js/03-code-quality/06-polyfills/article.md
Comment (odsantos) [inline 1-js/03-code-quality/06-polyfills/article.md]: Please,

- do not italicize the parentheses.

File: 1-js/03-code-quality/06-polyfills/article.md
Comment (odsantos) [inline 1-js/03-code-quality/06-polyfills/article.md]: Please

- do not italicize the parentheses.

File: 1-js/03-code-quality/06-polyfills/article.md
Comment (odsantos) [inline 1-js/03-code-quality/06-polyfills/article.md]: Please

- do not italicize the parentheses.

File: 1-js/03-code-quality/06-polyfills/article.md
Comment (odsantos) [inline 1-js/03-code-quality/06-polyfills/article.md]: Please

- do not italicize the parentheses.

File: 1-js/03-code-quality/06-polyfills/article.md
Comment (odsantos) [inline 1-js/03-code-quality/06-polyfills/article.md]: Please

- '... usar o Babel em produção (aplicações reais, não me fase de desenvolvimento) para tornar ...'

**Comment by odsantos** [comment]
Hi I hope to do it as soon as possible. Thanks.

**Comment by odsantos** [comment]
@luizbaldi Hi, would you please check this recent review? Thanks.

File: 1-js/03-code-quality/06-polyfills/article.md
Comment (odsantos) [reply 1-js/03-code-quality/06-polyfills/article.md]: Hi,

I do not know what you've done, since it appears that you changed where it was good and left where I asked for a change.
- Would you please revert that change at '... propostas à linguagem ...' , and consider my comment above?
Thanks.

File: 1-js/03-code-quality/06-polyfills/article.md
Comment (luizbaldi) [reply 1-js/03-code-quality/06-polyfills/article.md]: Sorry, I didn't get this one. What should be done here?

**Comment by luizbaldi** [comment]
Hi @odsantos, I'm sorry for the confusion. I hope things are correct now.

Let me know if any more change is needed, and thanks for your comprehension, I'm just trynna help here :)

File: 1-js/03-code-quality/06-polyfills/article.md
Comment (odsantos) [inline 1-js/03-code-quality/06-polyfills/article.md]: - 'postergar'
(didn't know it.)
:+1:

**Comment by luizbaldi** [comment]
@odsantos Sorry, it's being a while but I hope we can still finish this :)

**Comment by odsantos** [comment]
Hi @luizbaldi could you please resolve the conflicts? Thank you.

**Comment by luizbaldi** [comment]
@odsantos Conflicts solved :)

**Comment by odsantos** [comment]
@luizbaldi Hi, happy to see you're well. It's truly great  Please try to always be safe. .
It's really a while. Let's do hope we may get this complete.
I expect to check it soon.
A nice day to you and all close.

**Comment by luizbaldi** [comment]
@odsantos Thanks for your kind words, hope you're safe and doing well too. And no need to rush here, take your time :)

File: 1-js/03-code-quality/06-polyfills/article.md
Comment (odsantos) [inline 1-js/03-code-quality/06-polyfills/article.md:4]:
```suggestion
A linguagem JavaScript evolui constantemente. Novas propostas para a linguagem aparecem regularmente, onde elas são analisadas e, se consideradas que valem a pena, anexadas à lista em <https://tc39.github.io/ecma262/>, onde depois avançam para a [especificação](http://www.ecma-international.org/publications/standards/Ecma-262.htm).
```

File: 1-js/03-code-quality/06-polyfills/article.md
Comment (odsantos) [inline 1-js/03-code-quality/06-polyfills/article.md:14]:
```suggestion
Quando usamos funcionalidades modernas da linguagem, alguns interpretadores podem não suportar esse código. Como dito anteriormente, nem todas as funcionalidades são implementadas em todos os lugares.
```

File: 1-js/03-code-quality/06-polyfills/article.md
Comment (odsantos) [inline 1-js/03-code-quality/06-polyfills/article.md:22]:
```suggestion
1. Primeiro, o programa responsável pelo _transpile_. O desenvolvedor roda o programa na própria máquina, onde o código é reescrito para um padrão mais antigo. Após isso, o codigo é entregue ao website para os usuários. Sistemas de _build_ modernos como [webpack](http://webpack.github.io/) fornecem meios para executar o _transpiler_ automaticamente em todas as mudanças realizadas no código, de forma a tornar fácil de integrar em um processo de desenvolvimento.
```

File: 1-js/03-code-quality/06-polyfills/article.md
Comment (odsantos) [inline 1-js/03-code-quality/06-polyfills/article.md:27]:
```suggestion
    O _transpiler_ reescreve o código, transformando estruturas de sintaxe em outras mais antigas. Porém, para novas funcionalidades, precisamos de os implementar. O JavaScript é uma linguagem altamente dinâmica, onde _scripts_ podem não apenas adicionar novas funcionalidades, mas também modificar as já incorporadas na linguagem, de forma a que elas se comportem de acordo com o padrão moderno.
```

File: 1-js/03-code-quality/06-polyfills/article.md
Comment (odsantos) [inline 1-js/03-code-quality/06-polyfills/article.md:35]:
```suggestion
Então, precisamos de configurar o _transpiler_ e adicionar o _polyfill_ para fazer com que os interpretadores antigos suportem novas funcionalidades.
```

File: 1-js/03-code-quality/06-polyfills/article.md
Comment (odsantos) [inline 1-js/03-code-quality/06-polyfills/article.md:37]:
```suggestion
Se nos orientarmos por _engines_ modernos e só usarmos as _features_ (funcionalidades) suportadas em toda parte, não precisamos de usar o Babel.
```

File: 1-js/03-code-quality/06-polyfills/article.md
Comment (odsantos) [inline 1-js/03-code-quality/06-polyfills/article.md:42]:
```suggestion
Muitos exemplos são executáveis diretamente no local, como este:
```

**Review by odsantos — CHANGES_REQUESTED** [review]
Hi @luizbaldi ,

Here are my suggestions. I do not know if you agree.

Thank you.

**Comment by luizbaldi** [comment]
@odsantos It's impossible to get this merged because this file was reviewed infinite times and every time I make changes you came up with more things to be changed that were ignored in the past. Why don't you suggest everything at once so we can be more assertive here and get this done? It just feels pointless to keep making changes forever and this is not good for me nor you. I'll finish those last suggestions that you sent and I hope we can do this right.

**Comment by odsantos** [comment]
@luizbaldi Hi, I hope to check it later. Thank you.

**Review by odsantos — APPROVED** [review]
Hi @luizbaldi,

I believe we still need a second reviewer.

Thank  you very much.

**Comment by luizbaldi** [comment]
@odsantos I did everything I could to get this right and contribute to this project, but it's been over two months since you approved the changes and now we have a lot of conflicts based on the changes that were added to the original article after this "final approval".

I am closing this PR because, as mentioned before, it just feels pointless to keep making changes here 😞.

**Comment by odsantos** [comment]
Hi @luizbaldi,
It's sad you did that. Technology changes and we need to keep updating our skills.
Please, don't quit good deeds but bad ones.
Hope you may continue like before.
Would you try to be healthy and well?
Thank you.

**Pull request closed without merging**
